### PR TITLE
Changing api

### PR DESCRIPTION
**Pull request by asko328P**
Changed api, updated registration forms, still have to implement meaningful redirection

### CODE REVIEW

**Review by nikolamajmunovic — CHANGES_REQUESTED** [review]
remove forgot password
error handling 
set phone number in form
validate email
validate password(some regex for example min 8 char with numbers and one capital letter)

File: src/actions/auth.js [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
+        dispatch({
+          type: REGISTER_SUCCESS,
+        });
+        dispatch({
+          type: SET_MESSAGE,
+          payload: response.message,
+        });
+      }
+      if (response.code === 1) {
+        console.log('There is already an account registered with this email');
```
Comment (nikolamajmunovic) [inline src/actions/auth.js:19]: remove console.logs

File: src/services/auth.service.js [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
   };
   return loginPost();
 };
+const register = (first_name, last_name, email, password) => {
+  const registerPost = async () => {
+    const response = await axios.post(`${API_URL}register`, {
+      first_name,
+      last_name,
+      email,
+      phone_number: '+46934394409422754',
```
[the remark this answers is not in the document]
Comment (asko328P) [reply src/services/auth.service.js:20]: Inserted

File: src/actions/auth.js [hunk 3]
```diff
@@ -1,58 +1,58 @@
...
-
-     */
 };
 
 export const login = (username, password, setLoading) => {
   return async (dispatch) => {
     try {
       const response = await AuthService.login(username, password);
-      if (response.code === 0) {
+      if (response.success === true) {
```
[the remark this answers is not in the document]
Comment (asko328P) [reply src/actions/auth.js:46]: Added checks, for both register and login

File: src/actions/auth.js (hunk 1 above)
Comment (asko328P) [reply src/actions/auth.js:19]: All console.logs removed.

**Comment by asko328P** [comment]
Adjusted code, will push changes to branch tomorrow morning.

**Pull request merged**